### DISCUSSION TITLE
General Question

### DISCUSSION THREAD

**Discussion by SiiLeon** [visualon/featureide.net#110] [Q&A]
Hallo, 
ich würde gerne das featureIDE.net Projekt verwenden, aber ich weiß nicht wie man den Composer benutzt oder das Modell, die Konfiguration mit Features und ihren Klassen kombinieren soll.
Könnten Sie mir bitte helfen?

**Comment by viceice** [comment]
Da kann ich leider nicht helfen, wir nutzen nur die Feature Klassen.

Der ganze UI Krams ist nicht enthalten. Wir nutzen nur die Kernbibliothek.

**Reply by SiiLeon** [reply]
Dann kann man den Composer, zB. featureHouse, nicht verwenden ?

**Reply by viceice** [reply]
weiß ich nicht, ich weiß nicht was das ist.
